lib/modules/platform/azure/index.ts

```diff
@@ -156,10 +155,10 @@ export async function getRawFile(
       undefined,
       undefined,
       branchOrTag ? versionDescriptor : undefined,
+      true,
```

Comment:
what is this flag changing?

Reply:
it's the "includeContent" parameter, for including the content when getting the full item.